python/models/hierarchical_filtering_proxy_model.py

```diff
@@ -23,12 +26,12 @@ class HierarchicalFilteringProxyModel(QtGui.QSortFilterProxyModel):
     """
     
     
-    class _IndexAcceptedCache(object):
+    class _IndexRejectedCache(object):
         """
-        Cached 'accepted' values for indexes.  Uses a dictionary that maps a key to a tuple 
-        containing a QPersistentModelIndex for the index and its accepted value.
+        Cached 'rejected' values for indexes.  Uses a dictionary that maps a key to a tuple
```

Comment:
would be great to know what 'rejected' means? Perhaps a practical example of where this is being used could help.

Reply:
Will clarify.